docs/power-payment-app.md

```diff
@@ -0,0 +1,127 @@
+- Choronko Users: Individuals who can send and receive payments using the Choronko device without needing a smartphone.
+
+## Technologies used
+
+- ReactJS for building the user interface and implementing a Progressive Web App (PWA) approach.
+- HTML and CSS for the structure and styling of the user interface.
+- RESTful APIs for communication with the backend server.
+
+## Architecture and Interaction
+
+### User Interface:
+
+- **Registration Process**: The registration process is a series of steps that allows users to create an account and access the payment app's features. The steps typically include:
+
+    1. **Step 1 - Personal Information**: Users enter their personal details, such as name, phone number etc. This information is necessary for account creation and verification.
+
+    2. **Step 2 - Pin Setup**: Users set up a secure pin to protect their account. The pin should meet specific criteria, such as minimum length of numbers etc.
+
+    3. **Step 3 - OTP Confirmation**: Once users have entered their information, set up a pin, they are provide with an OTP in which they now confirm their account for security measures.
+
+    4. **Step 4 - Confirmation**: Once users have entered their information, set up a pin, etc . they can proceed to complete the registration process. The "Register" button is clicked to finalize the registration.
+
+    **note:** **The above registration method is for a smartphone user.**
+    **See mockup:** ![reference image](/mockups_wireframes/Registration_process.png)
+
+
+- **Registartion process for a choronko user**:As a choronko user who wants to register, all you need to do is :
+
+1. **Step 1 - Initiating Registration**: The choronko user can initiate the registration process by calling the powerpay service and then providing  their personal information such as full name , phone number etc.
+
+2. **Step 2 -  Mobile Number Verification**:After collecting the user's mobile number, the Choronko registration system will initiate a verification step. This will be done by sending an OTP (One-Time Password) to the user's mobile number via SMS.
+
+3. **Step 3 -  OTP Verification**:The Choronko user will be prompted to enter the OTP or verification code received via SMS. The system will validate the code to ensure the mobile number is associated with the user and proceed to the next step upon successful verification.
+
+4. **Step 4 -  Setting a Personal Identification Number (PIN)**: For security purposes, the Choronko user will be prompted to set a PIN .The system will guide the user through the process of setting a secure PIN following specific requirements.
+
+5. **Step 5 -  Completing Registration**: Once the user has set their PIN, the registration process for the Choronko user is considered complete. The system will store the user's information and PIN securely in the backend, associating it with the user's mobile number.
+
+
+
+## User Workflow
```

Comment:
In the workflow, you can now reuse the interfaces you defined earlier to explain better.

E.g: User registration:
In this process,... (basic description of the process)
1. A user enter his data in the screen (1), 
2. Then press next to go on screen (3) where he should enter the OTP he received from our backend 
3. At last, he is being redirected to screen (?) where he...